impala/tests/util.py

```diff
@@ -42,6 +43,11 @@ def __init__(self, host=None, port=None, hive_port=None, auth_mech=None):
         else:
             self.port = get_env_var('IMPYLA_TEST_PORT', int, 21050)
 
+        if http_port is not None:
+            self.http_port = port
```

Comment:
I think, this should be: 
self.http_port = http_port

Reply:
yes good catch